app.py

```diff
@@ -15,45 +15,62 @@ def index():
     # If form is submitted
     if request.method == 'POST':
         NotamRequest = Models.NotamRequest(request.form)
-
-        # get lat/long of airports
-        NotamRequest.startLat, NotamRequest.startLong = alc.get_lat_and_lon(NotamRequest.startAirport)
-        NotamRequest.destLat, NotamRequest.destLong = alc.get_lat_and_lon(NotamRequest.destAirport)
+        airports = [NotamRequest.startAirport, NotamRequest.destAirport]
+        #counts in the number of airports the user is travelling to
+        countAirports = 0

```

Comment:
no need to count airports one by one, can just set `countAirports = len( Notam.Request.destination )`